dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/bootstrap/DubboBootstrap.java

```diff
@@ -155,6 +155,8 @@
 
     private AtomicBoolean started = new AtomicBoolean(false);
 
+    private AtomicBoolean ready = new AtomicBoolean(true);
```

Comment:
Why is the default value not `false`?

Reply:
Sometimes user would not call the DubboBootstrap.start method, so then the app ready function would not working, then the app should ready in default!